budgeteer-web-interface/src/test/java/org/wickedsource/budgeteer/web/pages/budgets/edit/form/EditBudgetFormTest.java

```diff
@@ -74,7 +75,7 @@ void testSubmitValidInput() {
     @Test
     void testDuplicateImportKey() {
         FormTester formTester = tester.newFormTester("form", false);
-        doThrow(new DataIntegrityViolationException("contraints!")).when(budgetServiceMock).saveBudget(any());
+        doThrow(new InvalidBudgetImportKeyException()).when(budgetServiceMock).saveBudget(any());
```

Comment:
This is missing an a similar test for the InvalidBudgetNameException and the exception combining both issues.

Reply:
I implemented for each exception and error message one test.